packages/angular-sdk-components/src/lib/_components/field/user-reference/user-reference.component.ts

```diff
@@ -160,25 +173,24 @@ export class UserReferenceComponent implements OnInit, OnDestroy {
       const queryPayload = {
         dataViewName: OPERATORS_DP
       };
-      PCore.getRestClient()
-        .invokeRestApi('getListData', { queryPayload } as any, '') // 3rd arg empty string until typedef marked correctly
-        .then((resp: any) => {
-          const ddDataSource = resp.data.data.map(listItem => ({
-            key: listItem.pyUserIdentifier,
-            value: listItem.pyUserName
-          }));
-          this.options$ = ddDataSource;
-        })
-        .catch(err => {
-          console.log(err);
-        });
+      const resp: any = await PCore.getRestClient().invokeRestApi('getListData', { queryPayload } as any, ''); // 3rd arg empty string until typedef marked correctly
+      if (resp?.data) {
+        const ddDataSource = resp.data.data.map(listItem => ({
+          key: listItem.pyUserIdentifier,
+          value: listItem.pyUserName
+        }));
+        this.options$ = ddDataSource;
+      }
```

Comment:
Any reason behind changing Promise to async/await?

If you are going to use async/await, please use try catch to catch errors

Reply:
updated code with try catch block. Need to wait on the api call to finish and give us data to be used in _filter function, otherwise search box is displaying empty.